server.go

```diff
@@ -1754,6 +1754,20 @@ func (s *server) createLivenessMonitor(cfg *Config, cc *chainreg.ChainControl) {
 		chainHealthCheck, diskCheck, tlsHealthCheck,
 	}
 
+	outboundPeersCheck := healthcheck.NewObservation(
```

Comment:
This needs to be done in 2 PRs. One that has the changes for the `healthcheck` submodule (that needs to be merged first, then a new tag needs to be pushed), and one for the rest.